pkg/util/https/https.go

```diff
@@ -66,3 +66,15 @@ func LogRequest(logger *logrus.Entry, r *http.Request) *logrus.Entry {
 		WithField("headers", r.Header).
 		WithField("requestURI", r.RequestURI)
 }
+
+// RequestWrapper perform checks on request
+func RequestWrapper(h http.Handler) http.Handler {
```

Comment:
I might have been a bit quick to approve - I think this could go into main.go / move into th sdk package - since it's not a generic https utility that's used across the project.

But still like the approach!

Reply:
+1

What about just making this a function in `cmd/sdk-server/main.go` called something like `func healthcheckWrapper(...)` (not exported outside of that main pkg)?

Reply:
+1

@roberthbailey I agree with you.
We can put the wrapper function in `cmd/sdk-server/main.go`

Reply:
Sounds great. Let me know once it's ready for review, and we can try to get it merged before the release candidate tomorrow.

Reply:
I have made the suggested changes, but build is failing.